extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -14,53 +14,50 @@
  * limitations under the License.
  */
 
-.i-amphtml-story-desktop-fullbleed .i-amphtml-story-button-container,
-.i-amphtml-story-desktop-panels .i-amphtml-story-button-container {
+.i-amphtml-story-button-container {
+  cursor: pointer !important;
   position: absolute !important;
+  width: 30px !important;
   top: 0 !important;
   bottom: 0 !important;
-  left: 0 !important;
-  right: 0 !important;
+  height: calc(100vh - 150px) !important;
+  margin: auto 0 !important;
+  background: none !important;
+  transition: 150ms opacity linear, 150ms visibility linear !important;
+  outline: none !important;
   z-index: 100002 !important;
-  pointer-events: none !important;
-  display: block !important;
 }
 
-.i-amphtml-story-button-move,
-.i-amphtml-story-fwd-next > .i-amphtml-story-page-sentinel,
-.i-amphtml-story-back-prev > .i-amphtml-story-page-sentinel {
-  pointer-events: all !important;
+/* Hiding the outline on small screens since this button is intended to be 
+used with mobile screen readers such as TalkBack which add focus lines. */
+amp-story:not([desktop]) .i-amphtml-story-button-container {
+  pointer-events: none !important;
```

Comment:
if this is just meant to hide the outline, should this just be `outline: none`?

otherwise, we should update the comment to reflect why we don't want any pointer events at all

Reply:
I updated the comment. Thank you for catching this!